models/_sector/nft/trades/ethereum/platforms/superrare_ethereum_base_trades.sql

```diff
@@ -170,13 +170,13 @@ SELECT
     end as trade_type,
     case
         when a.seller = coalesce(minter.to, minter_superrare.to)
-        then cast(ROUND((0.03+0.15) * (a.price_raw)) as uint256) -- superrare takes fixed 3% fee + 15% commission on primary sales
-        else cast(ROUND(0.03 * (a.price_raw)) as uint256)    -- fixed 3% fee
+        then cast(ROUND((3E-2+15E-2) * (a.price_raw)) as uint256) -- superrare takes fixed 3% fee + 15% commission on primary sales
```

Comment:
nit: This might be easier to read a `0.03e0 + 0.15e0`

Reply:
@0xRobin gave feedback on using `double '0.03'` so I'll change this